Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -374,25 +374,25 @@ extension SendViewController {
         }
 
         let usdAmount = NSDecimalNumber(value: dcrAmount).multiplying(by: exchangeRate)
-        self.usdAmountLabel.text = "\(usdAmount.round(2)) USD"
+        self.usdAmountLabel.text = "\(usdAmount.round(2).formattedWithSeparator) USD"
```

Comment:
Should be 
```suggestion
        self.usdAmountLabel.text = "$\(usdAmount.round(2).formattedWithSeparator)"
```

Reply:
that is not the current design, USD is used here